src/backend/egl/context.rs

```diff
@@ -37,6 +40,36 @@ pub struct EGLContext {
 unsafe impl Send for EGLContext {}
 
 impl EGLContext {
+    /// Creates a new `EGLContext` from raw handles.
+    ///
+    /// # Safety
+    ///
+    /// This function is marked unsafe, because the context cannot be made current on another thread without
+    /// being unbound again (see [`EGLContext::unbind`]).
+    pub unsafe fn from_raw<L>(
```

Comment:
We need to explain some more requirements:

Does the display need to live for the lifetime of the returned object.
Does the config need to live for the lifetime of the context?
Does the context need to live for the lifetime of the object?

Reply:
From what I understand, display handle must outlive EGLDisplay, and config + context handles must outlive EGLContext. I don't know how to format safety docs for those though.

Reply:
Pretty much just some bullet points.